ramalama/console.py

```diff
@@ -0,0 +1,53 @@
+"""
```

Comment:
Please remove the license text

Reply:
still disagree here, specially in new files but okay.